project/cmdbuilder.go

```diff
@@ -0,0 +1,53 @@
+package project
+
+import (
+	"context"
+	"io"
+	"os"
+	"os/exec"
+)
+
+var X = 14
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported var X should have comment or be unexported